src/components/editor/property/LookupWithMultipleAuthorities.jsx

```diff
@@ -41,6 +41,8 @@ const LookupWithMultipleAuthorities = (props) => {
     return Object.values(authorities)
   }, [props.propertyTemplate.authorities])
 
+  const clearSearchResults = useCallback(() => dispatch(clearSearchResultsAction('resource')), [dispatch])
```

Comment:
Doesn't this clear the search results of the main search, not lookup?

Reply:
It appears to work on both.